content/kotlin/concepts/math-methods/terms/tanh/tanh.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.tanh()'
+Description: 'Computes the hyperbolic tangent of an angle.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.tanh()`** method in Kotlin's `math` class computes the hyperbolic tangent of an angle given in radians.
```

Comment:
Here, add a sentence stating that the method always takes in one parameter- the angle.